subscriptions/sales_flow/create_a_quotation.rst

```diff
@@ -0,0 +1,78 @@
+==============================================
+Create a quotation using subscription products
+==============================================
+
+Selling a digital product or service can give you instant gratification. However, you then have
+to work hard for a new customer to make a purchase. It costs time and money. By convincing customers
+to sign up for a subscription, you will maximize your income and streamline your cash flow. You can
+sell any type of product or service through a subscription business model.
+
+.. raw:: html
+
+   <div align="center" style="color:#AD5E99; font-size: 2rem ;margin: 20px 0"> <b>The only limit is
+   your imagination.</b> </div>
+
+Here is a scenario using **Odoo Subscriptions** to create a quotation including subscription
+products.
```

Comment:
The rest of the team usually uses italic for app names.
But, you can still do it bold if you prefer, I think.

Reply:
Yep, Keli told me that she uses bold for app names and italic for features... I am comfortable with that too. So I don't know... @jubodoo can I keep it this way?

Reply:
I don't care if we use italic or bold. But we should agree on something and use the same in all our docs because that's not the case for now.

Reply:
Agree with that :) @jubodoo